src/transformers/tokenization_utils_base.py

```diff
@@ -1594,7 +1594,7 @@ def __init__(self, **kwargs):
         self.model_input_names = kwargs.pop("model_input_names", self.model_input_names)
 
         # By default, cleaning tokenization spaces for both fast and slow tokenizers
-        self.clean_up_tokenization_spaces = kwargs.pop("clean_up_tokenization_spaces", True)
+        self.clean_up_tokenization_spaces = kwargs.pop("clean_up_tokenization_spaces", False)
```

Comment:
We need to throw a warning here, check th erest of the codebase for deprecation warning!